next/src/components/cards/FacilityCard.tsx

```diff
@@ -0,0 +1,67 @@
+import { Typography } from '@bratislava/component-library'
+import React, { useId } from 'react'
+
+import { ArrowRightIcon } from '@/src/assets/icons'
+import CardBase, { CardBaseProps } from '@/src/components/cards/CardBase'
+import { CardTitleLevel } from '@/src/components/cards/getCardTitleLevel'
+import ImagePlaceholder from '@/src/components/common/Image/ImagePlaceholder'
+import StrapiImage, { StrapiUploadImage } from '@/src/components/common/Image/StrapiImage'
+import MLink from '@/src/components/common/MLink/MLink'
+import cn from '@/src/utils/cn'
+import { CommonLinkProps } from '@/src/utils/getLinkProps'
+
+export type Props = {
+  title: string
+  linkProps: CommonLinkProps
+  cardTitleLevel?: CardTitleLevel
+  image?: StrapiUploadImage | null | undefined
+  imageSizes?: string
+} & CardBaseProps
+
+/**
+ * Figma: https://www.figma.com/design/17wbd0MDQcMW9NbXl6UPs8/DS--Component-library?node-id=17952-15653&m=dev
+ */
+
+const FacilityCard = ({
+  title,
+  cardTitleLevel = 'h3',
+  image,
+  imageSizes,
+  linkProps,
+  className,
+  ...rest
+}: Props) => {
+  const titleId = useId()
+
+  return (
+    <CardBase
+      variant="border"
+      className={cn('h-full rounded-lg bg-background-passive-base', className)}
```

Comment:
I don't usually see `h-full` in card itself. Instead, we usually add `*:h-full` to wrapping `<li>` element.